coordinator/src/main/java/org/apache/uniffle/coordinator/web/vo/ServerNodeVO.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.coordinator.web.vo;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Set;
+
+import org.apache.uniffle.common.ServerStatus;
+import org.apache.uniffle.coordinator.ServerNode;
+
+public class ServerNodeVO {
```

Comment:
Why do we need this class？

Reply:
> Why do we need this class？

Format the display of the interface, such as memory automatically converted according to the value and added units: B, KB, MB, GB. The time stamp is displayed in the format yyyy-MM-dd, which requires such a class to do processing.